language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java

```diff
@@ -577,9 +577,11 @@ protected LSCompletionItem getImplicitNewCItemForClass(ClassSymbol symbol, Balle
         return completionItems;
     }
 
-    private CompletionItem getModuleCompletionItem(String label, String insertText, @Nonnull List<TextEdit> txtEdits) {
+    private CompletionItem getModuleCompletionItem(String label, String insertText, @Nonnull List<TextEdit> txtEdits, 
+                                                   String prefix) {
```

Comment:
Shall we add the `Nonnull` annotation to this parameter? If not, let's add a null check for the safety.

Reply:
Added a null check